README.md

```diff
@@ -327,6 +338,164 @@ Default: `{{ __mssql_client_repository }}`
 
 Type: `string`
 
+### `mssql_ha_configure`
+
+Use the variables with the `mssql_ha_` prefix to configure SQL Server for high
```

Comment:
Use the variable with the 'mssql_ha_' prefix to configure a SQL Server Always On availability group to provide high availability.

Reply:
Thank you, applied